### PR TITLE
DT-780 finishing off a couple of TODO's around population of return types.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
Added new modal return type for service layer to populate and return for prison to prison moves.

Also reverted hsqldb dependency 2.5.0 to 2.4.1 as this causes test failures locally.  There is a ticket in the backlog to investigate and fix.

### CODE REVIEW

File: src/main/java/net/syscon/elite/api/model/ScheduledPrisonToPrisonMove.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.time.LocalDateTime;
+
+@ApiModel(description = "Represents the data for a scheduled a prison to prison move.")
```
Comment (brightonsbox) [inline src/main/java/net/syscon/elite/api/model/ScheduledPrisonToPrisonMove.java:14]: Meh... should read `for a scheduled prison to prison move` (no extra `a`)

File: src/test/java/net/syscon/elite/service/PrisonToPrisonMoveSchedulingServiceTest.java [hunk 2]
```diff
@@ -305,12 +310,6 @@ void schedule_move_errors_when_active_escort_agency_not_found() {
                 .hasMessage("Escort type PECS for prison to prison move not found.");
     }
 
-    @Disabled
-    @Test
-    void schedule_move_errors_when_existing_active_schedule() {
-        // TODO - DT-780
-    }
-
```
Comment (brightonsbox) [inline src/test/java/net/syscon/elite/service/PrisonToPrisonMoveSchedulingServiceTest.java:313]: Is this no longer relevant?

File: src/main/java/net/syscon/elite/api/model/ScheduledPrisonToPrisonMove.java (hunk 1 above)
Comment (matt-crowson-moj) [reply src/main/java/net/syscon/elite/api/model/ScheduledPrisonToPrisonMove.java:14]: Thanks.  Will correct.

File: src/test/java/net/syscon/elite/service/PrisonToPrisonMoveSchedulingServiceTest.java (hunk 2 above)
Comment (matt-crowson-moj) [reply src/test/java/net/syscon/elite/service/PrisonToPrisonMoveSchedulingServiceTest.java:313]: Not sure yet...

**Pull request merged**